src/components/map-marker/map-marker.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, Element, Event, EventEmitter, Prop } from "@stencil/core";
+import { IComponent } from "../common/interfaces";
+import {
+  icon,
+  marker
+  // tslint:disable-next-line:no-submodule-imports
+} from "leaflet/dist/leaflet-src.esm";
+
+@Component({
+  shadow: false,
+  styleUrl: "map-marker.scss",
+  tag: "gx-map-marker"
+})
+export class MapMarker implements IComponent {
+  @Element() element: HTMLElement;
+
+  private markerInstance: any;
+
+  /**
+   * The coordinates where the marker will appear in the map.
```

Comment:
We should clearly state the coordinates system that is being used for the map.

Reply:
Ready.